gaphor/ui/css_function_completion_provider.py

```diff
@@ -0,0 +1,100 @@
+from gi.repository import Gio, GObject, Gtk, GtkSource
```

Comment:
In Gaphor we tend to use file names of concatenated words, no undescore (e.g. `treecomponent.py`).

Reply:
So then this would be `cssfunctioncompletionprovider.py`, yuck!
Quite hard to read.

Reply:
Both CompletionProvider classes can be added to `csscompletion.py`.

Reply:
True!